endoreg_db/views/patient_finding_views.py

```
@@ -0,0 +1,130 @@
+from endoreg_db.models import (
+    FindingLocationClassificationChoice,
+    FindingMorphologyClassificationChoice
+)
+from rest_framework import serializers
+
+class PatientFindingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PatientFinding
+        fields = '__all__'
+
+class PatientFindingViewSet(ModelViewSet):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)